demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -314,19 +315,79 @@ def is_script_condition_branches_handled(self, task: Dict) -> bool:
         """
         is_all_condition_branches_handled: bool = True
         next_tasks: Dict = task.get('nexttasks', {})
-        if '#default#' not in next_tasks:
-            error_message, error_code = Errors.playbook_unhandled_condition(task.get('id'), {'else'})
-            if self.handle_error(error_message, error_code, file_path=self.file_path):
-                self.is_valid = is_all_condition_branches_handled = False
 
         if len(next_tasks) < 2:
             # there should be at least 2 next tasks, we don't know what condition is missing, but we know it's missing
-            error_message, error_code = Errors.playbook_unhandled_condition(task.get('id'), {})
+            error_message, error_code = Errors.playbook_unhandled_script_condition_branches(task.get('id'), {})
             if self.handle_error(error_message, error_code, file_path=self.file_path):
                 self.is_valid = is_all_condition_branches_handled = False
 
         return is_all_condition_branches_handled
 
+    def are_default_conditions_valid(self):  # type: () -> bool
+        """Check whether the playbook conditional tasks' default options are valid
+
+        Return:
+            bool. if the Playbook's handles all condition default options correctly.
+        """
+        default_conditions_valid: bool = True
+        tasks: Dict = self.current_file.get('tasks', {})
+        for task in tasks.values():
+            if task.get('type') == 'condition':
+                # builtin conditional task
+                if task.get('nexttasks'):
+                    default_conditions_valid = self.is_default_not_only_condition(
+                        task) and default_conditions_valid
+                # ask conditional task
+                if task.get('message') and task.get('message').get('replyOptions'):
+                    default_conditions_valid = self.is_default_not_only_reply_option(
+                        task) and default_conditions_valid
+        return default_conditions_valid
+
+    @error_codes('PB125')
+    def is_default_not_only_condition(self, task: Dict) -> bool:
+        """Checks whether the #default# is the only branch
+
+        Args:
+            task (dict): task json loaded from a yaml
+
+        Return:
+            bool. if the task's next tasks have more than just the default option.
+        """
+        is_default_not_only_condition_res: bool = True
+        next_tasks: Dict = task.get('nexttasks', {})
+
+        if len(next_tasks) == 1:
```

Comment:
Actually, it doesn't matter if the length of next_tasks is 1. The conditional task may have 3 next_tasks with "Else" label, which is also a problem. The check should verify whether the next_tasks have labels other than "Else". If not - error.

Reply:
https://github.com/demisto/demisto-sdk/pull/2497/commits/5e676671ffcd634df2b18811c0f217738525ed04

Reply:
I phrased myself completely wrong. I meant:
Doesn't matter the amount of next_tasks. If ALL of the next_tasks (whether 1 out of 1, or 3 out of 3) lead to "Else" - it should be an error, because we can't have paths only for "else"